### DISCUSSION TITLE
Disabling luxon for date columns

### DISCUSSION THREAD

**Discussion by maximelebastard** [adonisjs/core#4627] [Help] [answered]
Hi,

I'm trying to use Adonis/Lucid with AdminJS and the adonis adapter.

Unfortunately, AdminJS provides and consumes native JS Date columns when dealing with datetime inputs, which triggers type errors.

As Luxon is not useful in my case, the easiest would be to stop using it in my models - it would be ok for me to work with JS Dates as I always did.

Is there a simple way to do that ?

**Comment by RomainLanz** [comment, marked as answer]
Hey @maximelebastard! 👋🏻 

If you don't use the `@column.dateTime()` decorator you will be fine.

```ts
class User extends BaseModel {
  @column()
  createdAt: string
  
  @column({
    consume: (v: string) => new Date(v),
    prepare: (v: Date) => v.toISOString(), 
  )
  updatedAt: Date
}
```

**Reply by maximelebastard** [reply]
🙏 So reactive, thank you very much !
Should I propose a PR to make the documentation clearer about this ? It wasn't that clear for a beginner like me :)

**Reply by RomainLanz** [reply]
Yes, you can. Lucid documentation may need some love, so feel free to send any relevant PR. 👍🏻
